adding param to search (#23789)
demisto/content: Packs/SplunkPy/ReleaseNotes/3_0_3.md

#### Integrations
##### SplunkPy

> - Added the argument *fast_mode* to **splunk-search** command  for quicker results return.

Reviewer: Add here also the docker image update please

> - Added a new parameter **fast_mode** to **splunk-search** command  for quicker results return.

Reviewer: ```suggestion
- Added the argument *fast_mode* to **splunk-search** command  for quicker results return.
```
